library/js/prescription-autocomplete/prescription-autocomplete.js

```diff
@@ -0,0 +1,138 @@
+/*
+ *  @package   OpenEMR
+ *  @link      http://www.open-emr.org
+ *  @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c )2019. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ *
+ */
+
+
+
+$(document).ready(function () {
```

Comment:
this is now illegal (sentence involves floating on an iceberg for several months). to avoid this sentence, use this instead:
```
$(function () {
```

Reply:
I would ask but some times ignorance is bliss.